ktxspec.adoc

```diff
@@ -959,26 +959,26 @@ The ETC1S image format is indicated by `colorModel` `KDF_DF_MODEL_ETC1S`
 (= 163) together with `vkFormat` `VK_FORMAT_UNDEFINED` (= 0). The
 DFD must be as described in Section 5.6.11 _KHR_DF_MODEL_ETC1S_ of
 <<KDF13>>. Because ETC1S does not support an alpha component, Basis
-Universal uses 2 _slices_, (_planes_ in DFD-speak).
+Universal uses 2 _slices_, (_planes_ in DFD-speak) to represent RGBA images.
 This color model provides the following channel ids that must be used to
 indicate the use of a slice.
 +
 [width=100%,align=center,cols="<30,^10,<60",options=header]
 |===
-| Id | Value | Type 
-|`KHR_DF_CHANNEL_ETC1S_RGB` | 0  | 3 component: opaque color. RGB
-                                   components in the rgb channels.
-|`KHR_DF_CHANNEL_ETC1S_RRR` | 3  | 1 component: R component replicated
-                                   in all 3 rgb channels for better
-                                   compression results.
-|`KHR_DF_CHANNEL_ETC1S_GGG` | 4  | Not used independently. 1 component:
-                                   G component replicated in all 3 rgb
-                                   channels
-|`KHR_DF_CHANNEL_ETC1S_AAA` | 15 | Not used independently. 1 component:
-                                   alpha component replicated in all 3
-                                   rgb channels.
+| Id | Value | Type
+|`KHR_DF_CHANNEL_ETC1S_RGB` | 0  | 3 components: opaque color. RGB
+                                   components in the rgb slice channels.
+|`KHR_DF_CHANNEL_ETC1S_RRR` | 3  | 1 component: R component in the r
```

Comment:
The meaning of "r slice channel" isn't clear.  Do you mean in the slice's r channel? Should we use component instead of channel?

Reply:
> Do you mean in the slice's r channel?

Yes. What would be a better term for it?

Reply:
Either "slice's r channel" or "slice's r component" is fine.

Reply:
Switched to using "component" to not confuse with DFD's channels.